app/controllers/reservations_controller.rb

```diff
@@ -4,7 +4,7 @@ class ReservationsController < ApplicationController
 
   customer_tab  :all
   before_action :authenticate_user!, except: [:index]
-  before_action :check_acting_as, only: [:switch_instrument, :show, :list]
+  before_action :check_acting_as, only: [:switch_instrument, :list]
```

Comment:
Yes. The `:check_acting_as` was blocking admins ordering on behalf of users from hitting the show action, which is what the calendar icon links to for its download.